specification/purview/Azure.Analytics.Purview.DataMap/models.tsp

```diff
@@ -0,0 +1,1963 @@
+  failedImportInfoList?: ImportInfo[];
+
+  @doc("successful importInfoList")
+  successImportInfoList?: ImportInfo[];
+}
+
+@doc("ImportInfo")
+model ImportInfo {
+  @doc("childObjectName")
+  childObjectName?: string;
+
+  @doc("importStatus")
+  importStatus?: ImportStatus;
+
+  @doc("parentObjectName")
+  parentObjectName?: string;
+
+  @doc("remarks")
+  remarks?: string;
+}
+
+@doc("MoveEntitiesOptions")
+model MoveEntitiesOptions {
+  @doc("An array of entity guids to be moved to target collection.")
+  entityGuids?: string[];
+}
+
+@doc("The glossary object.")
+model AtlasGlossary {
+  ...AtlasGlossaryBaseObject;
+
+  @doc("An array of categories.")
+  categories?: AtlasRelatedCategoryHeader[];
+
+  @doc("The language of the glossary.")
+  language?: string;
+
+  @doc("An array of related term headers.")
+  terms?: AtlasRelatedTermHeader[];
+
+  @doc("The usage of the glossary.")
+  usage?: string;
+}
+
+@doc("The header of the related category.")
+model AtlasRelatedCategoryHeader {
+  @doc("The GUID of the category.")
+  categoryGuid?: string;
+
+  @doc("The description of the category header.")
+  description?: string;
+
+  @doc("The display text.")
+  displayText?: string;
+
+  @doc("The GUID of the parent category.")
+  parentCategoryGuid?: string;
+
+  @doc("The GUID of the relationship.")
+  relationGuid?: string;
+}
+
+@doc("The header of the related term.")
+model AtlasRelatedTermHeader {
+  @doc("The description of the related term.")
+  description?: string;
+
+  @doc("The display text.")
+  displayText?: string;
+
+  @doc("The expression of the term.")
+  expression?: string;
+
+  @doc("The GUID of the relationship.")
+  relationGuid?: string;
+
+  @doc("The status of term relationship.")
+  status?: AtlasTermRelationshipStatus;
+
+  @doc("The steward of the term.")
+  steward?: string;
+
+  @doc("The GUID of the term.")
+  termGuid?: string;
+}
+
+@doc("The glossary base object.")
+model AtlasGlossaryBaseObject {
+  ...AtlasBaseModelObject;
+
+  @doc("An array of classifications.")
+  classifications?: AtlasClassification[];
+
+  @doc("The long version description.")
+  longDescription?: string;
+
+  @doc("The name of the glossary object.")
+  name?: string;
+
+  @doc("The qualified name of the glossary object.")
+  qualifiedName?: string;
+
+  @doc("The short version of description.")
+  shortDescription?: string;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "It is an Atlas model"
+  @doc("ETag for concurrency control.")
+  lastModifiedTS?: string;
+
+  @doc("The created time of the record.")
+  @encode(DateTimeKnownEncoding.unixTimestamp, int64)
+  createTime?: utcDateTime;
+
+  @doc("The user who created the record.")
+  createdBy?: string;
+
+  @doc("The update time of the record.")
+  @encode(DateTimeKnownEncoding.unixTimestamp, int64)
+  updateTime?: utcDateTime;
+
+  @doc("The user who updated the record.")
+  updatedBy?: string;
+}
+
+@doc("The base model object.")
+model AtlasBaseModelObject {
```

Comment:
Why does this object exist? I don't see it being used other than spread into AtlasGlossaryBaseObject. It most likely should either be an alias or not exist.

Reply:
Rename to alias. Thanks